hera_sim/beams.py

```diff
@@ -193,15 +195,56 @@ def _efield_eval(
         # Set beam Jones matrix values (see Eq. 5 of Kohn+ arXiv:1802.04151)
         # Axes: [phi, theta] (az and za) / Feeds: [n, e]
         # interp_data shape: (Naxes_vec, Nspws, Nfeeds or Npols, Nfreqs, Naz)
-        interp_data = modulate_with_dipole(
-            az_grid, za_grid, f_grid, self.ref_freq, beam_values, fscale
-        )
+        if self.polarized:
+            interp_data = modulate_with_dipole(
+                az_grid, za_grid, f_grid, self.ref_freq, beam_values, fscale
+            )
+        else:
+            # Empty data array
+            interp_data = np.zeros(
+                (2, 2) + f_grid.shape, dtype=np.complex128
+            )
+            interp_data[1, 0, :, :] = beam_values  # (theta, n)
+            interp_data[0, 1, :, :] = beam_values  # (phi, e)
```

Comment:
This has a very strange property that used to be present in some pyuvsim analytic beams that I think is extremely unphysical: The North feed only responds to zenith angle oriented polarized light and the East feed only responds to azimuthally-oriented polarized light. Are you sure this is what you want it to do?

I'd expect an unpolarized beam to respond equally to zenith-angle oriented and azimuthally oriented polarized light.